components/account/AccountHeader.vue

```diff
@@ -115,7 +115,7 @@ const personalNoteMaxLength = 2000
     <div p4 mt--18 flex flex-col gap-4>
       <div relative>
         <div flex justify-between>
-          <button shrink-0 :class="{ 'rounded-full': !isSelf, 'squircle': isSelf }" p1 bg-base border-bg-base z-2 @click="previewAvatar">
+          <button shrink-0 class="h-full" :class="{ 'rounded-full': !isSelf, 'squircle': isSelf }" p1 bg-base border-bg-base z-2 @click="previewAvatar">
```

Comment:
This can directly be `h-full`, no?
```suggestion
          <button shrink-0 h-full :class="{ 'rounded-full': !isSelf, 'squircle': isSelf }" p1 bg-base border-bg-base z-2 @click="previewAvatar">
```

Reply:
Ah I forgot that was possible in this project, you're totally right

Reply:
adjusted it 😄